### PR TITLE
Ca refactor/table component on synchronizers

### PR DESCRIPTION
**Pull request by cagodoy**
Screen Shot 2023-03-26 at 05 45 56

### CODE REVIEW

File: app/pkg/synchronizer/synchronizer.ts [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
-    try {
-      const url = `${this.URL}/api/v1/events`;
-      const res = await fetch(url, {
-        method: "GET",
-        headers: {
-          "content-type": "application/json",
-        },
-      });
+    const url = `${this.baseURL}/api/v1/events`;
+    return await Fetch<ListEventsResponse>("GET", url);
```
Comment (nicobarra) [inline app/pkg/synchronizer/synchronizer.ts:15]: 👍  it looks better

File: app/pkg/synchronizer/synchronizer.ts [hunk 2]
```diff
@@ -66,3 +70,7 @@ export default class Synchronizer {
     }
   }
 }
+
+// stop event
```
Comment (nicobarra) [inline app/pkg/synchronizer/synchronizer.ts]: This comments should be removed

**Review by nicobarra — APPROVED** [review]
Mint 🌱

File: app/pkg/synchronizer/synchronizer.ts (hunk 2 above)
Comment (cagodoy) [reply app/pkg/synchronizer/synchronizer.ts]: ok

**Pull request merged**